### ISSUE TITLE
Bubble Indicator cut off on the leftmost data point in case displayYAxis = true

### ISSUE THREAD

**Issue by angel1st** [aeyrium/bezier-chart#42]
In case `displayYAxis = true` and the user scrolls to the leftmost chart side, the bubble indicator gets cut off on the left side. See the screenshot enclosed.
![Screenshot_1566035965](https://user-images.githubusercontent.com/8547131/63216151-ee7d5a80-c139-11e9-81c3-6ee0705f1616.png)

In case yAxis isn't present all seems good. Apparently, some space should be reserved for the bubble, in case yAxis is present.

**Comment by diegoveloper** [comment]
Yeah , I know this bug, but since the chart is horizontal scrollable I can't find the way to fix this. Pull requests are welcome
